src/data/onChainPoolDataViaReadContract.ts

```diff
@@ -1,8 +1,20 @@
-import { PublicClient, parseAbi, Address, Abi, Hex } from 'viem';
+import { PublicClient, Address, Abi, Hex } from 'viem';
 import { getPoolAddress } from '../utils';
 import { OnChainPoolData } from './enrichers/onChainPoolDataEnricher';
 import { SwapOptions } from '../types';
 
+import ComposableStablePoolV5Abi from '../abi/abis/ComposableStablePoolV5Abi.json';
+import ConfigurableRightsPoolAbi from '../abi/abis/ConfigurableRightsPoolAbi.json';
+import ConvergentCurvePoolAbi from '../abi/abis/ConvergentCurvePoolAbi.json';
+import FxPoolAbi from '../abi/abis/FxPoolAbi.json';
+import GyroEV2Abi from '../abi/abis/GyroEV2Abi.json';
+import LiquidityBootstrappingPoolAbi from '../abi/abis/LiquidityBootstrappingPoolAbi.json';
+import ManagedPoolAbi from '../abi/abis/ManagedPoolAbi.json';
+import MetaStablePoolAbi from '../abi/abis/MetaStablePoolAbi.json';
+import PhantomStablePoolAbi from '../abi/abis/PhantomStablePoolAbi.json';
+import VaultAbi from '../abi/abis/VaultAbi.json';
+import WeightedPoolAbi from '../abi/abis/WeightedPoolAbi.json';
```

Comment:
It would be nice if you could move these abis to the parent folder and export them on index.ts, so it's easier to import them elsewhere.

Reply:
included